modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/DefaultExecutionMediator.java

```diff
@@ -59,7 +61,7 @@
      * @return the operation's result
      * @throws Exception if the operation or a {@link Interceptor#before(OperationContext)} invokation fails
      */
-    public Object execute(OperationExecutor executor, OperationContext context) throws Exception
+    public Object execute(OperationExecutor executor, OperationContext context, ExceptionEnricher exceptionEnricher) throws Exception
```

Comment:
The ExceptionEnricher should be part of the mediator's state since it's always going to be the same one